test/preparation/test_clean.py

```diff
@@ -18,7 +18,8 @@
 sp = PDBParser()
 
 def test_remove_solvent():
-    pdb_file_path = f"{DATA_DIR}1Q4T_solvent_cofactor_2chain.pdb"
+    pdb_file_path = f"{DATA_DIR}1Q4T_ligand_test.pdb"
+    print(pdb_file_path)
```

Comment:
Please change the path to 1Q4T_cofactor_2chain_not_from_1.pdb and remove the print

Reply:
Also 1Q4T_ligand_test.pdb will be a duplicate file of 1Q4T_cofactor_2chain_not_from_1.pdb